src/sqlparser/src/ast/statement.rs

```diff
@@ -218,9 +218,12 @@ impl Parser {
         if connector.contains("-cdc") {
```

Comment:
Recommend parsing all possible values (mongodb-cdc, postgres-cdc, mysql-cdc) to `enum` and use equal conditions here. This code looks too lossy and it's hard to know what will happen if I pass some strange values end with `-cdc`

Reply:
leave to the PR that adding e2e tests.